RealmSwift/Realm.swift

```diff
@@ -22,6 +22,16 @@ import Realm.Private
 
 #if swift(>=3.0)
 
+private let defaultQueue: DispatchQueue = {
+    let attributes: DispatchQueue.GlobalAttributes
+    if #available(OSXApplicationExtension 10.10, *) {
+        attributes = DispatchQueue.GlobalAttributes.qosBackground
+    } else {
+        attributes = DispatchQueue.GlobalAttributes.priorityBackground
```

Comment:
Xcode warns that we're using a deprecated symbol here, and I'm not sure how to silence it. I could define the queue from Objective-C (where this was not deprecated?) and use it from Swift, but that seems gross.